src/tests/ftest/pool/evict_test.py

```diff
@@ -50,8 +50,7 @@ def connected_pool(self, hostlist, targets=None):
             TestPool (object)
 
         """
-        pool = TestPool(self.context, self.log,
-                        dmg_command=self.get_dmg_command())
+        pool = TestPool(self.context, self.get_dmg_command(), log=self.log)
```

Comment:
The `log` argument is defunct for TestPool.create():
```suggestion
        pool = TestPool(self.context, self.get_dmg_command())
```